apps/docs/spec/cli_v1_config.yaml

```diff
@@ -1112,6 +1180,17 @@ parameters:
       - name: 'Auth Server configuration'
         link: 'https://supabase.com/docs/reference/auth'
 
+  - id: 'auth.third_party.aws_cognito'
```

Comment:
can consider `<provider>` syntax, but let's wait for verdict of [this internal discussion](https://supabase.slack.com/archives/C023E4L60R3/p1733205517257259)